libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -1333,6 +1333,39 @@ public static Task<List<VideoCaptureFormat>> GetVideoCaptureFormatsAsync(string
             });
         }
 
+        /// <summary>
+        /// Frame height round mode.
+        /// </summary>
+        /// <seealso cref="SetFrameHeightRoundMode(FrameHeightRoundMode)"/>
+        public enum FrameHeightRoundMode
+        {
+            /// <summary>
+            /// Leave frames unchanged.
+            /// </summary>
+            NONE = 0,
+            /// <summary>
+            /// Crop frame height to the nearest multiple of 16.
+            /// </summary>
+            CROP = 1,
+            /// <summary>
+            /// Pad frame height to the nearest multiple of 16.
+            /// </summary>
+            PAD = 2
+        }
+
+        /// <summary>
+        /// Use this function to select whether resolutions where height is not multiple of 16
+        /// should be cropped, padded or left unchanged.
+        /// Default is <see cref="FrameHeightRoundMode.NONE"/>, except on Hololens 1 where it is
+        /// <see cref="FrameHeightRoundMode.CROP"/> to avoid severe artifacts produced by
+        /// the H.264 hardware encoder.  
```

Comment:
That doesn't seem to be the case currently. `InitHololensH264Workaround()`, which overrides the default no-op behavior, is only called from within `SetFrameHeightRoundMode()`. So by default the no-op behavior is still active.

Reply:
Good catch. There was a call in the PeerConnection factory method, but it seems it has been merged out 😕